app/routes/company/components/CompaniesPage.js

```diff
@@ -3,65 +3,78 @@
 import React from 'react';
 import styles from './Company.css';
 import LoadingIndicator from 'app/components/LoadingIndicator';
+import InfiniteScroll from 'react-infinite-scroller';
 import { Link } from 'react-router';
 import { Image } from 'app/components/Image';
 import type { Company } from 'app/models';
-import { Button } from 'app/components/Form';
 
 type Props = {
   companies: Array<Company>,
+  fetchMore: () => void,
   showFetchMore: () => void,
-  fetchMore: () => void
+  hasMore: Boolean
 };
 
-export function CompanyItem({ company }: any) {
+const CompanyItem = function({ company }: any) {
+  let websiteString = company.website.split('/');
+  websiteString = websiteString[0] + '//' + websiteString[2];
+
   return (
     <div className={styles.companyItem}>
-      <div>
+      <div className={styles.companyItemContent}>
         <Link to={`/companies/${company.id}`}>
-          <h3 className={styles.companyItemTitle}>{company.name}</h3>
+          <div className={styles.companyLogo}>
+            {<Image src={company.logo} />}
+          </div>
+          <div className={styles.companyItemTitle}>
+            <h2>{company.name}</h2>
+          </div>
+          <div className={styles.companyInfo}>
+            <span>
+              {company.joblistingCount > 0 &&
+                company.joblistingCount + ' jobbannonser'}
+            </span>
+            <span>
+              {company.eventCount > 0 && company.eventCount + ' arrangementer'}
+            </span>
+          </div>
         </Link>
-        <a href={company.website}>
-          <div className={styles.website}>{company.website}</div>
-        </a>
-        <div>{company.companyType}</div>
-        <div>{company.address}</div>
+        {company.website &&
+          !company.website.includes('example.com') && (
+            <a href={company.website}>
```

Comment:
Maybe add a `target="_blank"` so it opens in a new tab

Reply:
Good idea!